sdk/search/search/review/search.api.md

```diff
@@ -67,6 +290,34 @@ export interface GetDocumentOptions<Fields> extends OperationOptions {
     selectedFields?: Fields[];
 }
 
+// @public (undocumented)
+export type GetIndexOptions = OperationOptions;
+
+// @public (undocumented)
+export type GetIndexStatisticsOptions = OperationOptions;
+
+// @public
+export interface GetIndexStatisticsResult {
+    readonly documentCount: number;
+    readonly storageSize: number;
+}
+
+// @public
+export interface Index {
+    analyzers?: AnalyzerUnion[];
```

Comment:
Should this be `customAnalyzers: CustomAnalyzer[]`? That's the sense I get reading the docs anyway.

Reply:
No, because custom analyzers are not the only analyzers you can add to an index. There are also pre-defined analyzers that are configurable, like `PatternAnalyzer` for example.